docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -963,10 +963,10 @@ must be present in the `tasks` array.
 
 ### Stepback
 
-Stepback is set to true if you want to stepback and test earlier commits
-in the case of a failing task. This can be set or unset at the
-top-level, at the build variant level, and for individual tasks (in the task definition or for the
-task within a specific build variant).
+Stepback is set to true if you want to stepback and test earlier commits in the case
+of a failing task (a failing task does not include system failed, setup failed, timed out, etc.).
```

Comment:
"normally failing task" since the others technically are failures as well, they're just more specific failure types.